src/EnergyPlus/MixedAir.cc

```diff
@@ -747,8 +750,19 @@ namespace MixedAir {
                     bool CoolingActive = false;
                     Real64 OAUCoilOutTemp = 0.0;
                     bool ZoneEquipFlag = false;
-                    OutsideAirSys(OASysNum).compPointer[CompIndex]->simulate(
-                        CompName, FirstHVACIteration, AirLoopNum, CompIndex, HeatingActive, CoolingActive, CompIndex, OAUCoilOutTemp, ZoneEquipFlag);
+                    Real64 sensOut = 0.0;
+                    Real64 latOut = 0.0;
+                    OutsideAirSys(OASysNum).compPointer[CompIndex]->simulate(CompName,
```

Comment:
@Myoldmopar some of the variables that are needed to call the UnitarySystem simulate function are not available in the MixedAir module (same problem in OutdoorAirUnit.cc and SimAirServingZones.cc). So declared them here. I wonder if it's better to do this at the top of the function and declare as "something const VarName(xxxx)"? Also, is there any speed impact for declaring them here?

Reply:
The performance hit here will probably be undetectable, but it would probably be best to declare them as static const if those are able to be constant.

Reply:
However, I'm glad you are thinking about it.